.github/PULL_REQUEST_TEMPLATE/dependencies.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
There's the potential for confusion where feature work also involves adding or updating a dependency (eg. do I need 2 prs? if so, how do they get applied so said feature is testable?) so I'd be inclined to say that splitting out to a separate template isn't merited.